scripts/variantstore/wdl/GvsCreateVAT.wdl

```diff
@@ -13,7 +16,7 @@ workflow GvsSitesOnlyVCF {
         String table_suffix
 
         String? service_account_json_path
-        File? gatk_override
+        File? gatk_override ## TODO do you think we can drop this altogether? when would we need this?
```

Comment:
I say remove it — if you've never needed it, might as well rip it out (we can always add it back in if useful).